src/components/FindMovie/FindMovie.tsx

```diff
@@ -17,41 +68,64 @@ export const FindMovie: React.FC = () => {
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={query}
+              onChange={event => {
+                setQuery(event.target.value);
+                setHasNoSuchTitle(false);
```

Comment:
move this to separate function and call it here 
onChange={handleInput}
and in upper scope 
const handleInput = (event: React.ChangeEvent<HTMLSelectElement>) => {
 setQuery(event.target.value);
setHasNoSuchTitle(false);
}